### PR TITLE
[GCU] add ipv6, ethernet interface to kvm test suite

### PR DESCRIPTION
**Pull request by isabelmsft**
### Description of PR

Summary:
Add previous Generic Configer Updater ipv6 and ethernet interface config tests into kvm test suite
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Move GCU test into kvm daily test
#### How did you do it?
Add GCU test case into kvmtest.sh
#### How did you verify/test it?
Run test of sonic-mgmt/tests/generic_config_updater/*.py on KVM

### CODE REVIEW

File: tests/generic_config_updater/test_eth_interface.py [hunk 1]
```diff
@@ -144,6 +144,9 @@ def test_toggle_pfc_asym(duthost, ensure_dut_readiness, pfc_asym):
 
 @pytest.mark.parametrize("fec", ["rs", "fc"])
 def test_replace_fec(duthost, ensure_dut_readiness, fec):
+    if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
+        pytest.skip("Not supported on virtual switch")
```
Comment (wangxin) [inline tests/generic_config_updater/test_eth_interface.py:148]: Is it better to skip using the conditional mark?

Comment (qiluo-msft) [reply tests/generic_config_updater/test_eth_interface.py:148]: @wangxin could you share an example?

Comment (wangxin) [reply tests/generic_config_updater/test_eth_interface.py:148]: Conditional mark is a plugin for adding marks based on some customized conditions. For example, add `skip` mark if platform is `x86_64-kvm_x86_64-r0`. By using this plugin, tests can be skipped in earlier stage to save some overhead of preparing fixtures.
Document is here: https://github.com/Azure/sonic-mgmt/tree/master/tests/common/plugins/conditional_mark
Examples are here: https://github.com/Azure/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

Please ensure that the added entries are in alphabetic order, so it is easier for people to read the code and the chance of adding duplicated entries is low.

Comment (isabelmsft) [reply tests/generic_config_updater/test_eth_interface.py]: thank you, I changed to conditional mark

**Review by qiluo-msft — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**